src/app/RecordingList/ActiveRecordingsList.tsx

```diff
@@ -192,3 +183,97 @@ export const ActiveRecordingsList: React.FunctionComponent<ActiveRecordingsListP
     </RecordingsDataTable>
   </>);
 };
+
+export interface RecordingActionsProps {
+  isOpen: boolean;
+  setOpen: (open: boolean) => void;
+  index: number;
+  recording: Recording;
+}
+
+export const RecordingActions: React.FunctionComponent<RecordingActionsProps> = (props) => {
+  const context = React.useContext(ServiceContext);
+  const notifications = React.useContext(NotificationsContext);
+  const [grafanaEnabled, setGrafanaEnabled] = React.useState(false);
+  const [uploadIds, setUploadIds] = React.useState([] as string[]);
+
+  React.useEffect(() => {
+    const sub = context.commandChannel.grafanaDatasourceUrl()
```

Comment:
I think this would be on the container-jfr side, but the jfr-datasource supports liveness on it's root url. Maybe a quick http test can be used so if someone does supply a url but it doesn't actually work, we still have it disabled? Does that make sense to you? I can file an issue and maybe look at it too for once

Reply:
Yea, makes sense to me. I've been meaning to implement some kind of `/health` for container-jfr itself for a while, so maybe what we can do is have that return an HTTP status to indicate its own health, and the response body can be some JSON that includes details on whether the datasaource or dashboard URL etc. are configured and are reachable from ContainerJFR as well.